src/__test__/Calculator.test.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-undef */
```

Comment:
- [ ] Kindly remove this eslint disable rule. It's best to fix the issue in the code.